src/ocean/io/stream/Format.d

```diff
@@ -138,7 +127,14 @@ class FormatOutput(T) : OutputFilter
 
         final FormatOutput format (Const!(T)[] fmt, ...)
         {
-            return format(fmt, _arguments, _argptr);
+            this.convert(&this.emit, _arguments, _argptr, fmt);
+            return this;
+        }
+
+        /// Used by derived classes to avoid deprecations - Remove in v4.0.0
+        final void doNotUse (Const!(T)[] fmt, TypeInfo[] arguments, ArgList args)
```

Comment:
Also, create an issue not to forget and maybe give it a name that says what it does (besides making it obvious it shouldn't be used).

Reply:
I already raised a P.R. against v4.x.x to remove it. Also the comment says why it's called this way. However, it should be `protected`